src/app.js

```diff
@@ -35,6 +35,17 @@ render(
   </MuiThemeProvider>, 
   document.getElementById('app'));
 
+function safeGetUserSettings() {
+    const redirect = (err) => {
+        window.location.assign(config.loginUrl);
```

Comment:
@tokland we have this redirection to loginUrl here and in redirectToLogin method. The url path and the code that makes the redirection is redundant. Can we maybe extract this redirection method into an utils function? Then in the sessiondialog, redirectToLogin doesn't need to be a method hardcoded but a property that we pass to the component SessionDialog.  What do u think? Maybe I am being too picky, if so just let me know :)